### ISSUE TITLE
Healing should optionally start at 0

### ISSUE THREAD

**Issue by Rycochet** [encounterplus/encounterplus#354]
It's nice to see how much extra damage something has taken when it's down, but when you then heal them back up the 5e rules start at 0, so even a single point of healing will get them up. Previous versions didn't do this, so it should be an option for people who like the old way (and running other systems etc).

**Comment by rmhoag** [comment]
Agreed. It's one extra step to do the math to add enough healing to bring them to 0, then add the healing applied on top of that.
